### DISCUSSION TITLE
Does generated GUID include empty guid?

### DISCUSSION THREAD

**Discussion by satuday** [AutoFixture/AutoFixture#1279] [Q&A] [answered]
Hi does generated GUID from autofixture include empty guid(00000000-0000-0000-0000-000000000000)?

**Comment by aivascu** [comment, marked as answer]
@satuday the `GuidGenerator` uses `Guid.NewGuid()`  (see [implementation](https://github.com/AutoFixture/AutoFixture/blob/730b47884975d0c1256209073a719a9fe510d8a9/Src/AutoFixture/GuidGenerator.cs#L17)) which is [guaranteed](https://docs.microsoft.com/en-us/dotnet/api/system.guid.newguid?view=net-5.0#remarks) to never return an empty `Guid`.

**Reply by satuday** [reply]
great, thank.
